pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -187,6 +187,48 @@ def test_to_datetime_format_weeks(self, cache):
 
 class TestToDatetime(object):
 
+    def test_to_datetime_now(self):
+        # See GH#18666
+        with tm.set_timezone('US/Eastern'):
+            npnow = np.datetime64('now').astype('datetime64[ns]')
+            pdnow = pd.to_datetime('now')
+            pdnow2 = pd.to_datetime(['now'])[0]
+            # These should all be equal with infinite perf
```

Comment:
give a little room, I have seen flakiness with these kinds of tests, maybe ``< 1e7`` or something

Reply:
Confused.  You want more room or less room?  1e7 < 1e10.

Reply:
nvm. you are comparing for 10s here. maybe add a comment.